client/src/common/parsec.ts

```diff
@@ -39,18 +47,22 @@ export async function listAvailableDevices(): Promise<Array<AvailableDevice>> {
   return await libparsec.listAvailableDevices(window.getConfigDir());
 }
 
+function getClientConfig(): ClientConfig {
+  return {
+    configDir: window.getConfigDir(),
+    dataBaseDir: window.getDataBaseDir(),
+    mountpointBaseDir: window.getMountpointDir(),
```

Comment:
should it not be: `mountpointDir: window.getMountpointDir(),` or `mountpointBaseDir: window.getMountpointBaseDir(),`?

Reply:
It's just that `ClientConfig` is defined by the bindings so I don't really have any control over it. If it was my choice I would indeed rename it to `mountpointDir`.